### ISSUE TITLE
Multiple entries: Rethinking tag voting

### ISSUE THREAD

**Issue by Shiroizu** [VocaDB/vocadb#1616]
Tag votes are not particularly useful, and incorrect tags are inconvenient to remove for regular users (through entry reports).

1) Only track the first tag vote on entries, other votes should be discarded. Sort tags by tag popularity: Rarest tags will be shown first in each category. 
2) Extend the tag removal permissions to regular users. Migrate functionality from /ManageTagUsages/ to the tag dialog.
3) When a tag is removed, send a notification to whoever added the tag with dispution instructions if the original tagger doesn't agree.
4) Track tag removals on users pages "100 tags added, 20 removed" or "tags: 80" with detailed breakdowns on hover. Users with a negative tag count would not be able to remove more tags.

**Comment by jukkahyv** [comment]
Maybe this is something you can trial? Personally, I use the tag votes quite often with genres. I guess ideally there should be some difference between "objective" tags like copyrights, and more subjective, like moods or genres.

**Comment by saturclay** [comment]
I think tag votes are useful for more tags with less clear-cut use cases, like genre and subjective tags. Maybe one idea would be to make it more of an upvote/downvote system, where any user could increase or decrease a tag's count by 1 for a given entry. That way, tags that only have one vote could easily be removed without sending an entry report (with the general assumption that tags are more likely to be accurate if more people have voted on them).

**Comment by riipah** [comment]
I agree. I see value in the voting for more subjective tags, and use it frequently myself. Maybe the tags could be separated to subjective (moods and genres) and objective/factual (copyrights etc.). A bit similar to locked tags on Nico.
